pynest/nest/lib/hl_api_types.py

```diff
@@ -686,8 +685,7 @@ def get(self, keys=None, output=''):
 
         # Return empty tuple if we have no connections or if we have done a
         # nest.ResetKernel()
-        num_conn = GetKernelStatus('num_connections')
-        if self.__len__() == 0 or num_conn == 0:
+        if self.__len__() == 0 or sli_func('GetKernelStatus /num_connections get') == 0:
```

Comment:
why was that necessary?

Reply:
this allows the removal of one import and solves the problem of circular imports